src/testing/Calculator.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
- [ ] You do have not any tests simulating user interactions, like clicking a button and the accompanying assertions. Kindly add tests using the React Testing Library to simulate user actions, like performing a calculation, clicking navbar links, etc.

  You can visit [this tutorial](https://www.robinwieruch.de/react-testing-library) for hints on how to do this.

Reply:
@TedLivist We have done this work.